ops/solana/solana.go

```diff
@@ -137,28 +138,67 @@ func (d *Deployer) DeployLINK() error {
 }
 
 func (d *Deployer) DeployOCR() error {
-	fmt.Println("Initializing OCR Feed...")
+	fmt.Println("Deploying OCR Feed:")
+	fmt.Println("Step 1: Init Requester Access Controller")
 	err := d.gauntlet.ExecCommand(
-		"ocr2:initialize:flow",
+		"access_controller:initialize",
+		d.gauntlet.Flag("network", d.network),
+	)
+	if err != nil {
+		return errors.New("AC initialization failed")
+	}
+	report, err := d.gauntlet.ReadCommandReport()
+	if err != nil {
+		return err
+	}
+	d.States[RequesterAccessController] = report.Responses[0].Contract
+
+	fmt.Println("Step 2: Init Billing Access Controller")
+	err = d.gauntlet.ExecCommand(
+		"access_controller:initialize",
+		d.gauntlet.Flag("network", d.network),
+	)
+	if err != nil {
+		return errors.New("AC initialization failed")
+	}
+	report, err = d.gauntlet.ReadCommandReport()
+	if err != nil {
+		return err
+	}
+	d.States[BillingAccessController] = report.Responses[0].Contract
+
+	input := map[string]interface{}{
+		"minAnswer":   "0",
+		"maxAnswer":   "10000000000",
+		"decimals":    9,
+		"description": "Test",
+	}
+
+	jsonInput, err := json.Marshal(input)
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("Step 3: Init OCR 2 Feed")
+	err = d.gauntlet.ExecCommand(
+		"ocr2:initialize",
 		d.gauntlet.Flag("network", d.network),
-		d.gauntlet.Flag("description", "ETH/USD"),
-		d.gauntlet.Flag("decimals", "8"),
-		d.gauntlet.Flag("maxAnswer", "100000"),
-		d.gauntlet.Flag("minAnswer", "0"),
+		d.gauntlet.Flag("requesterAccessController", d.States[RequesterAccessController]),
+		d.gauntlet.Flag("billingAccessController", d.States[BillingAccessController]),
 		d.gauntlet.Flag("link", d.States[LINK]),
+		d.gauntlet.Flag("input", string(jsonInput)),
 	)
 	if err != nil {
 		return errors.New("feed initialization failed")
 	}
 
```

Comment:
Consider adding (executing) the full set of Gauntlet OCR2 commands here in the Deployer, to serve us currently as an E2E Gauntlet OCR2 acceptance test.

Potentially set and update the config once, add inspection ...

Also, please add docs and `TODOs` that this should be moved from the Deployer to a proper Gautlet E2E test. cc @tateexon